src/components/blocks/layout/Header.tsx

```diff
@@ -51,17 +98,38 @@ export const Header = ({ data }: { data: PageBlocksHeader }) => {
 			)}
 		>
 			<div className='flex max-w-7xl justify-between items-center w-full p-4'>
-				{data.logo && data.logo.src && data.logo.alt && (
-					<Image src={data.logo?.src} alt={data.logo.alt} width={200} height={60} />
-				)}
+				<div className='flex items-center'>
+					{data.logo && data.logo.src && data.logo.alt && (
+						<Image src={data.logo?.src} alt={data.logo.alt} width={200} height={60} />
+					)}
+				</div>
 				{data.navLinks && (
-          <>
-            <Group className='md:w-1/2 flex justify-between' visibleFrom='sm'>
-              {items}
-            </Group>
-            <Burger opened={opened} onClick={toggle} size="sm" hiddenFrom="sm" />
-          </>
+					<>
+						<Group className='md:w-1/2 flex justify-between' visibleFrom='sm'>
```

Comment:
Instead of `justify-between` you should use `justify-around px-4` so that the elements space evenly and are separated from the edges evenly too 

![Screenshot 2024-11-13 at 10 44 46 AM](https://github.com/user-attachments/assets/63d75b9a-75f2-4f4e-99cd-b77f8d2eeaea)

After that it'll be good to merge - would need to make it responsive later tho